src/qtt/pgeometry.py

```diff
@@ -195,12 +195,12 @@ def list_objects(objectype=None, objectclassname='__123', verbose=1):
 
 def package_versions(verbose: int = 1):
     """ Report package versions installed """
-    print('numpy.__version__ %s' % numpy.__version__)
+    print('numpy.__version__ %s' % numpy)
```

Comment:
This does not print the version any longer right? (only the package location). I think numpy.__version__ should be available

Reply:
You're right, I removed it by mistake. I pushed a new commit that restores it.